styles/scss/utils/_variable.scss

```diff
@@ -36,19 +36,43 @@ $itemMinHeightMd: 30rem;
 $sideBarWidth: 15rem;
 $mobileSpacing: 1.5rem;
 $borderRadius: 0.8rem;
-
+$borderRadiusSm: 0.6rem;
+$hoverColor: #f6efee;
+$selectedColor: #fee6e3;
 
 $mobileWidth: 28rem;
 $tabletWidth: 42rem;
 $appSideBarWidth: 30.5rem;
 $minComputerWidth: 62.5rem;
-$maxListWidth: 80rem;
+$maxListWidth: 91.47rem;
```

Comment:
Why use non-integer values? I noticed that even when converted to pixels, the values are not integers. What is the purpose of this?

Reply:
I want 800px, but it couldn't be expelled, so I took an approximation.
Now I change it to 800px.